### PR TITLE
CSP-1200 Added length restriction for LinkedIn url

### PR DESCRIPTION
**Pull request by ManishAgarwalDfE**

### CODE REVIEW

File: src/SFA.DAS.Employer.Aan.Web/Validators/EditContactDetail/SubmitContactDetailModelValidator.cs [hunk 1]
```diff
@@ -6,9 +6,14 @@ namespace SFA.DAS.Employer.Aan.Web.Validators.EditContactDetail;
 public class SubmitContactDetailModelValidator : AbstractValidator<SubmitContactDetailModel>
 {
     public const string ShowLinkedinUrlValidationMessage = "You cannot select display check box without a value";
+    public const string AlphaNumericCharactersRegex = "^[a-zA-Z0-9]*$";
+    public const string LinkedinUrlLengthValidationMessage = "Your custom URL must contain 3-100 letters or numbers";
+    public const string LinkedinUrlPatternValidationMessage = "Your custom URL must not use spaces, symbols or special characters";
     public SubmitContactDetailModelValidator()
     {
         RuleLevelCascadeMode = CascadeMode.Stop;
+        RuleFor(x => x.LinkedinUrl).NotNull().WithMessage(LinkedinUrlLengthValidationMessage).Length(3, 100).WithMessage(LinkedinUrlLengthValidationMessage).Matches(AlphaNumericCharactersRegex)
```
Comment (MarkFCain) [inline src/SFA.DAS.Employer.Aan.Web/Validators/EditContactDetail/SubmitContactDetailModelValidator.cs:15]: can you align this to be more readable please? ta

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.Employer.Aan.Web/Validators/EditContactDetail/SubmitContactDetailModelValidator.cs (hunk 1 above)
Comment (DevenKShah) [inline src/SFA.DAS.Employer.Aan.Web/Validators/EditContactDetail/SubmitContactDetailModelValidator.cs:15]: Can this be aligned properly to make it readable

Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Employer.Aan.Web/Validators/EditContactDetail/SubmitContactDetailModelValidator.cs]: Done

Comment (ManishAgarwalDfE) [reply src/SFA.DAS.Employer.Aan.Web/Validators/EditContactDetail/SubmitContactDetailModelValidator.cs]: Done

**Review by MarkFCain — APPROVED** [review]

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**